flight/Modules/VibrationAnalysis/vibrationanalysis.c

```diff
@@ -52,7 +51,15 @@
 // Private constants
 
 #define MAX_QUEUE_SIZE 2
-#define STACK_SIZE_BYTES (200 + 484 + (13*fft_window_size)*0) // The fft memory requirement grows linearly 
+
+/*
+    Stack usage reported by gcc:
+    VibrationAnalysisCleanup     16
+    VibrationAnalysisInitialize  24
+    VibrationAnalysisStart       48
+    VibrationAnalysisTask        192
+*/
+#define STACK_SIZE_BYTES (192 + 48 + 16 + (2*3*window_size)*0) // The memory requirement grows linearly 
```

Comment:
Of course is not correct... 
I will look into this. Probably first using the code in the simulator target, and then in a FC.